x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -19838,6 +19838,7 @@
     "xpack.ingestPipelines.form.savePip10mbelineError.showFewerButton": "Masquer {hiddenErrorsCount, plural, one {# erreur} many {# erreurs} other {# erreurs}}",
     "xpack.ingestPipelines.form.savePipelineError.processorLabel": "Processeur {type}",
     "xpack.ingestPipelines.form.savePipelineError.showAllButton": "Afficher {hiddenErrorsCount, plural, one {# autre erreur} many {# autres erreurs} other {# autres erreurs}}",
+    "xpack.ingestPipelines.indexTemplatesList.viewButtonLabel": "Afficher",
```

Comment:
Please remove the translations added to `fr-FR.json`, `ja-JP.json` and `zh-CN.json`. That is typically handled by a separate team and will be updated at a later time.